libbeat/_meta/fields.yml

```diff
@@ -45,3 +45,36 @@ fields:
       description: >
         Contains user configurable fields.
 
+- key: cloud
+  title: Cloud Provider Metadata
+  description: >
+    Metadata from cloud providers added by the add_cloud_metadata processor.
+  fields:
+
+    - name: cloud.provider
+      example: ec2
+      description: >
+        Name of the cloud provider. Possible values are ec2, gce, or digitalocean.
+
+    - name: cloud.instance_id
```

Comment:
After further thought, I changed instance_type to machine_type (follow GCE's naming). So I don't think we need to make it `instance.id` now.